packages/app/src/debug/DebugErrored.vue

```diff
@@ -0,0 +1,40 @@
+<template>
+  <Alert
+    :title="t('debugPage.incomplete')"
+    status="warning"
+    :icon="ErrorOutlineIcon"
+    class="flex flex-col mb-24px w-full"
+  >
+    <div class="ml-5px">
+      <ul
+        v-for="(error, index) in errors"
+        :key="index"
+        class="list-disc ml-25px"
+      >
+        <li><pre>{{ error }}</pre></li>
```

Comment:
Probably outside the scope of this PR, but if you end up with a very long error the alert body ends up with an overflow, and when you scroll the background color of the alert box is messed up. Add a `{viewportWidth: 600}` to one of the `DebugRunStates` test cases for this component to see what I mean

Reply:
yeah I think I'll leave this for now - we're still figuring out what this error state will look like in practice. They shouldn't be very long